src/layouts/index.js

```diff
@@ -229,6 +229,7 @@ export default class TemplateWrapper extends Component {
               src="//js.hs-scripts.com/2235598.js"
             />
           )}
+          <meta name="description" content="Investor tools and market data portals / Advanced features - simple set-up">
```

Comment:
This is great - just fix the typo at the end here :) 
```suggestion
          <meta name="description" content="Investor tools and market data portals / Advanced features - simple set-up" />
```

Reply:
Haha, ops. Very sweet of you to call wonky html-ing “a typo”. 🤪